compiler/src/yul/mappers/contracts.rs

```diff
@@ -40,6 +40,15 @@ pub fn contract_def(
 
         let runtime = runtime::build_with_abi_dispatcher(context, stmt);
 
+
+        let data = if let Some(attributes) = context.get_contract(stmt) {
```

Comment:
Not sure if this is how you envisioned it. I collect static strings on the `scope`, then they end up on the `ContractAttributes` and then they can be used here.